src/coreclr/vm/dacenumerablehash.inl

```diff
@@ -46,8 +46,8 @@ DacEnumerableHashTable<DAC_ENUM_HASH_ARGS>::DacEnumerableHashTable(Module *pModu
     S_SIZE_T cbBuckets = S_SIZE_T(sizeof(VolatileEntry*)) * (S_SIZE_T(cInitialBuckets) + S_SIZE_T(2));
 
     m_cEntries = 0;
-    PTR_VolatileEntry* pBuckets = (PTR_VolatileEntry*)(void*)GetHeap()->AllocMem(cbBuckets);
-    ((size_t*)pBuckets)[0] = cInitialBuckets;
+    DPTR(PTR_VolatileEntry) pBuckets = (DPTR(PTR_VolatileEntry))(void*)GetHeap()->AllocMem(cbBuckets);
+    ((size_t*)pBuckets)[SLOT_LENGTH] = cInitialBuckets;
```

Comment:
It this one is compiled into the DAC, it's a bit off - `*dac_cast<PTR_DWORD>(pBuckets[SLOT_LENGTH]) = cInitialBuckets`; that being said - mutating methods in the DAC are not common.

Reply:
Right, ctor and GrowTable  are always running in-proc.